addons/vz99/CfgMagazines.hpp

```diff
@@ -0,0 +1,83 @@
+class CfgMagazines {
+    class 8Rnd_82mm_Mo_shells;
+
+    //HE
+    class GVAR(HE): 8Rnd_82mm_Mo_shells {
+        picture = QUOTE(PATHTOF(ui\shell_ca.paa));
+        author = "PabstMirror";
+        count = 1;
+        mass = 20;
+
+        GVAR(base) = QGVAR(HE);
+        displayName = "vz99 60mm HE Impact Fuze";
+        descriptionShort = "60mm HE Shell<br/>For use in the vz99 mortar<br/>With a basic impact fuze";
+        displayNameShort = "HE-IMP";
+        ammo = QGVAR(ammo_he);
+        initSpeed = 110;
+        scope = 2;
+    };
+    class GVAR(HE_charge0): GVAR(HE) {
+        initSpeed = 70;
+        scope = 1;
+    };
+
+    //HE w/ multi fuze
+    class GVAR(HE_multi): GVAR(HE) {
+        GVAR(base) = QGVAR(HE_multi);
+        displayName = "vz99 60mm HE Multi Fuze";
+        descriptionShort = "60mm HE Shell<br/>For use in the vz99 mortar<br/>ith a multi-option fuze (Impact, Near Surface, Proximity)";
+        displayNameShort = "HE-IMP";
+        initSpeed = 110;
+        GVAR(fuze) = "";
+    };
+    class GVAR(HE_multi_charge0): GVAR(HE_multi) {
+        initSpeed = 70;
+    };
+    class GVAR(HE_PRX): GVAR(HE_multi) {
```

Comment:
This should have a different display/description right? Indicating that it's an HE Proximity round, not a HE Multi round?